deploy/kube-templates/glusterfs-daemonset.yaml

```diff
@@ -104,3 +107,8 @@ spec:
       - name: glusterfs-ssl
         hostPath:
           path: "/etc/ssl"
+parameters:
+- name: TCMU_LOG_DIR
+  displayName: Tcmu runner log directory
+  description: This value is to set tcmu runner log directory
+  value: /var/log/glusterfs/gluster-block
```

Comment:
Kubernetes has no notion of Templates, that's an OpenShift thing, thus the `parameters` field is invalid. Please remove.